### PR TITLE
docs: move react-native docs to separate section

### PR DESCRIPTION
**Pull request by ctavan**

### CODE REVIEW

**Comment by ctavan** [comment]
The error message increased bundlesize slightly. I tried to reduce the error message as much as possible but still had to increase the bundlesize limits a tiny little bit.

Fine for you @broofa ?

**Comment by ctavan** [comment]
Alternatively we could also shorten the error message to just a link and explain the rest in the readme.

I guess that would be even better.

File: src/rng-browser.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ var rnds8 = new Uint8Array(16); // eslint-disable-line no-undef
 export default function rng() {
   if (!getRandomValues) {
     throw new Error(
-      'uuid: This browser does not seem to support crypto.getRandomValues(). If you need to support this browser, please provide a custom random number generator through options.rng.',
+      'uuid: This environment does not seem to support crypto.getRandomValues(). Install a polyfill or provide a custom random number generator through options.rng. For React Native, see https://github.com/uuidjs/uuid#react-native',
```
Comment (broofa) [inline src/rng-browser.js]:
```suggestion
      'getRandomValues() not supported. See https://github.com/uuidjs/uuid#getrandomvalues-not-supported.'
```

File: README_js.md [hunk 2]
```diff
@@ -325,6 +324,12 @@ Available bundles:
 - https://unpkg.com/uuid@latest/dist/umd/
 - https://cdn.jsdelivr.net/npm/uuid@latest/dist/umd/
 
+## React Native
```
Comment (broofa) [inline README_js.md]:
```suggestion
## "getRandomValues() not supported"
```

File: README_js.md [hunk 3]
```diff
@@ -325,6 +324,12 @@ Available bundles:
...
+
+If you are using this library from React Native, you need to add a polyfill for
+`crypto.getRandomValues()`. We recommend using
+[`react-native-get-random-values`](https://github.com/LinusU/react-native-get-random-values#readme).
```
Comment (broofa) [inline README_js.md]:
```suggestion
This error occurs in environments where the standard [crypto.getRandomValues()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues) API is not supported.  This issue can be resolved by adding an appropriate polyfill:
| Environment | Suggested Polyfill |
|---|---|
| React Native | [`react-native-get-random-values`](https://github.com/LinusU/react-native-get-random-values#readme) |
```

File: README_js.md [hunk 4]
```diff
@@ -25,6 +25,7 @@ For the creation of [RFC4122](http://www.ietf.org/rfc/rfc4122.txt) UUIDs
   - Node 8, 10, 12
   - Chrome, Safari, Firefox, Edge, IE 11 browsers
   - Webpack and rollup.js module bundlers
+  - [React Native](#react-native)
```
Comment (broofa) [inline README_js.md]:
```suggestion
  - React Native ([see note](#getrandomvalues-not-supported))
```

**Comment by ctavan** [comment]
@broofa applied your changes and removed the bundlewatch sizelimit change.

**Pull request merged**